apps/wallet-mobile/src/features/Swap/common/ConfirmRawTx/ConfirmRawTxWithOs.tsx

```diff
@@ -22,7 +25,7 @@ export const ConfirmRawTxWithOs = ({onConfirm}: {onConfirm?: (rootKey: string) =
     return (
       <View style={styles.center}>
         <Text style={styles.errorMessage} numberOfLines={3}>
-          {error.message}
+          {getErrorMessage(error, strings)}
```

Comment:
I'd rather having a `const errorMessage = ...` and in the component {errorMessage} just for readability (nitpicky) feel free to ignore it.